tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
-    "target": "es2022"
+    "target": "es2022",
+    "resolveJsonModule": true
```

Comment:
Not sure what this was added for? Do your tests work without it? If so, I'd remove it.